pkg/scheduler/internal/heap/heap_test.go

```diff
@@ -97,12 +141,17 @@ func TestHeap_Add(t *testing.T) {
 // TestHeap_Delete tests Heap.Delete and ensures that heap invariant is
 // preserved after deleting items.
 func TestHeap_Delete(t *testing.T) {
-	h := New(testHeapObjectKeyFunc, compareInts)
+	h := NewWithRecorder(testHeapObjectKeyFunc, compareInts, new(testMetricRecorder)) // cover conditions on metric recorder
```

Comment:
Do you want to add some statements to cover the metric inc/dec?

Reply:
Resolved in 1c12a991112